extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -184,7 +184,7 @@ class AmpLightbox extends AMP.BaseElement {
     this.initialize_();
```

Comment:
This changes the event bubbling behavior from `bubble` to `capture`. Was this intentional? Why is it needed?  Will it break usages of lightbox anywhere? 0.0

Reply:
Ah that was a holdover from when I was using `this.listen_(this.element.ownerDocument, ...)` since I think the document object is not included in the bubble phase, but it is included in capture. I changed it to listen to `documentElement` and didn't update the capture flag.